contracts/token/ERC20BridgedPermit.sol

```diff
@@ -0,0 +1,35 @@
+// SPDX-FileCopyrightText: 2024 OpenZeppelin, Lido <info@lido.fi>
+// SPDX-License-Identifier: GPL-3.0
+
+pragma solidity 0.8.10;
+
+import {ERC20Bridged} from "./ERC20Bridged.sol";
+import {ERC20Permit} from "./ERC20Permit.sol";
+
+contract ERC20BridgedPermit is ERC20Bridged, ERC20Permit {
```

Comment:
:warning: :warning: :warning: 
https://github.com/lidofinance/lido-improvement-proposals/blob/develop/LIPS/lip-10.md

Let's check if we need to initialize new storage (if any) and if everything is compatible with the old one
I.e., there should be a smooth transition

Reply:
might affect the contracts that already deployed behind proxies and needed to be upgraded with this PR